elasticache/variables.tf

```diff
@@ -21,3 +22,18 @@ variable "engine_version" {
   type    = string
   default = "6.x"
 }
+
+variable "snapshot_retention_limit" {
+  type    = number
+  default = 0
+}
+
+variable "replicas_per_node_group" {
```

Comment:
let's also add that to the variables in the stack module

Reply:
https://github.com/dbl-works/terraform/pull/86/commits/56eaed9766df5325d71c292952862ae151799590

Reply:
https://github.com/dbl-works/terraform/pull/86/commits/56eaed9766df5325d71c292952862ae151799590